pkg/console/subresource/consoleserver/config.go

```diff
@@ -0,0 +1,65 @@
+package consoleserver
```

Comment:
I've opened a [PR](https://github.com/openshift/library-go/pull/454) thats adding all this types to `library-go` so it can be shared across our repos.

Reply:
Yeah if we can put it somewhere that makes sense. Def realized not OpenShift/API.